### DISCUSSION TITLE
As a non-coder, How to host model-viewer.min.js yourself on the same server as your site?

### DISCUSSION THREAD

**Discussion by Shanty3546** [google/model-viewer#4210] [Q&A]
I had to sign up to GitHub just to ask this, I read in 'How should I access <model-viewer>?' part of the documentation that this is the best approach but I don't know the exact steps I need to take to be able to do it.
I am not a coder and that's why I don't know exactly what to do, if anyone can tell me exact steps to it would be amazing. I have a WordPress website and I use elementor on it.
I know this is super easy for most of you so please help me out.

**Comment by elalish** [comment]
For wordpress, I wouldn't bother - using one of our CDNs should be just fine. Hosting it yourself is more for a big commerce website that needs absolute minimum loading times and has their own systems for ensuring uptime.

**Reply by Shanty3546** [reply]
ok but if I copy paste the snippet in a html block, it shows like this and no model comes up, what to do?
![Screenshot 2023-04-12 022853](https://user-images.githubusercontent.com/130441105/231286900-6d6f1f1f-aedb-4f81-a9f1-da2e66fb67e8.png)

**Reply by elalish** [reply]
Try modifying our glitch: https://glitch.com/edit/#!/model-viewer and get that working first. Check your JS console for errors - I'm not sure wordpress is set up to serve 3D models.

**Reply by Shanty3546** [reply]
I don't know coding so I don't know how to modify html properly, if these are simple few steps then please tell me what to do, there are plugins that are able to post 3d but I liked the model-viewer's way of customization

**Reply by elalish** [reply]
Well, model-viewer's customization is done by modifying HTML, so it sounds like now is a good time to learn! Give Glitch a try and maybe take an online tutorial on HTML/CSS.

**Reply by Shanty3546** [reply]
bruh XD literally contradicting the makers of this platform, I figured it out myself and yes, it was something super small just like I knew, all it took was an expert to take a look and I did it. Giving such bad advice to learn the language to just make a small change ...that too in the times of chatgpt. I understand experts r always needed but that's not what I asked for and in my case u really gave such bad advice. If anyone get's stuck in some similar problem, please let me know.. I will help as the best of my abilities.

**Reply by SyedNisaruddin** [reply]
so what was the solution?
